ui-admin/src/util/tableUtils.test.tsx

```diff
@@ -9,11 +9,15 @@ import {
   getSortedRowModel,
   useReactTable
 } from '@tanstack/react-table'
-import { basicTableLayout, checkboxColumnCell, ColumnVisibilityControl } from './tableUtils'
+import { basicTableLayout, checkboxColumnCell, ColumnVisibilityControl, DownloadControl } from './tableUtils'
 import userEvent from '@testing-library/user-event'
 
-/** simple table with filters and a show/hide column control */
-const TestTableComponent = ({ initialValue }: {initialValue: ColumnFiltersState}) => {
+const DEFAULT_INITIAL_DATA = [{ consented: true, name: 'Fred' }, { consented: false, name: 'James' }]
```

Comment:
nit: default to me implies it would be empty [].

```suggestion
const SAMPLE_INITIAL_DATA = [{ consented: true, name: 'Fred' }, { consented: false, name: 'James' }]
```

Reply:
Done in 10e6f428